### PR TITLE
Tickets/dm 14044

### PR DESCRIPTION
**Pull request by gcomoretto**
Hi, I think this was the last bit of the release note missing.
Can you please review it?

### CODE REVIEW

File: known-issues.rst [hunk 1]
```diff
@@ -61,19 +64,24 @@ RHEL 7.*
...
 macOS specific
 --------------
 
+- Macs must use the :command:`clang` compiler, not :command:`gcc`.
+  :jirab:`DM-3405`
+
+  One version of this problem occurs when using Macports_, which, by
+  default, will create a symlink from :file:`/opt/local/bin/c++` to its
+  version of :command:`g++`. Try removing that, starting a new shell, and
+  restarting :command:`eups distrib install`.
```
Comment (jdswinbank) [inline known-issues.rst]: I _think_ this is no longer a problem, or — at least — is now easier to fix than forcing the user to either remove GCC or to delete symlinks.

```
$ sudo port install gcc7
--->  Computing dependencies for gcc7
--->  Fetching archive for gcc7
--->  Attempting to fetch gcc7-7.3.0_0.darwin_17.x86_64.tbz2 from https://packages.macports.org/gcc7
--->  Attempting to fetch gcc7-7.3.0_0.darwin_17.x86_64.tbz2.rmd160 from https://packages.macports.org/gcc7
--->  Installing gcc7 @7.3.0_0
--->  Activating gcc7 @7.3.0_0
--->  Cleaning gcc7
--->  Updating database of binaries
--->  Scanning binaries for linking errors
--->  No broken files found.

$ /opt/local/bin/g++
bash: /opt/local/bin/g++: No such file or directory

$ sudo port select gcc mp-gcc7
Selecting 'mp-gcc7' for 'gcc' succeeded. 'mp-gcc7' is now active.

$ /opt/local/bin/g++ --version
g++ (MacPorts gcc7 7.3.0_0) 7.3.0
Copyright (C) 2017 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

$ sudo port select gcc none
Selecting 'none' for 'gcc' succeeded. 'none' is now active.

$ /opt/local/bin/g++ --version
bash: /opt/local/bin/g++: No such file or directory
```
In other words:
* Installing GCC via Macports does not make it the default compiler;
* If it does happen to be set as the default compiler, `sudo port select gcc none` will revert that selection without the user needing to address the filesystem directly.

**Comment by jonathansick** [comment]
Draft is: https://pipelines.lsst.io/v/DM-14044/known-issues.html

File: known-issues.rst [hunk 2]
```diff
@@ -61,19 +64,24 @@ RHEL 7.*
...
+
 .. _installation-issues-macos:
 
 macOS specific
 --------------
 
+- Macs must use the :command:`clang` compiler, not :command:`gcc`.
+  :jirab:`DM-3405`
+
+  One version of this problem occurs when using Macports_, which, by
```
Comment (jonathansick) [inline known-issues.rst]: "MacPorts" not "Macports" (may be moot given John's comment below)

File: known-issues.rst [hunk 3]
```diff
@@ -53,6 +53,9 @@ Red Hat (and clones) specific
   Attempting to use MKL will cause the test suite to fail, automatically
   aborting the installation. :jirab:`DM-5105`
 
+- Ensure that the :command:`gcc` compiler version supports **C++14** as 
+  specified in :ref:`Pre-requisites <source-install-redhat-prereqs>`.
```
Comment (jonathansick) [inline known-issues.rst]: Sentence works better as

```rst
specified in the :ref:`pre-requisites <source-install-redhat-prereqs>`.
```

File: known-issues.rst (hunk 1 above)
Comment (gcomoretto) [reply known-issues.rst]: If it is not a problem, I would suggest to remove the section.  If this requires specific installation, should not be added in the prereq page?

File: known-issues.rst [hunk 4]
```diff
@@ -61,19 +64,24 @@ RHEL 7.*
...
 
+- Macs must use the :command:`clang` compiler, not :command:`gcc`.
+  :jirab:`DM-3405`
+
+  One version of this problem occurs when using Macports_, which, by
+  default, will create a symlink from :file:`/opt/local/bin/c++` to its
+  version of :command:`g++`. Try removing that, starting a new shell, and
+  restarting :command:`eups distrib install`.
+
 macOS 10.12 (Sierra) and OS X 10.11 (El Capitan)
```
[the remark this answers is not in the document]
Comment (gcomoretto) [reply known-issues.rst]: Yes, 10.13

File: known-issues.rst [hunk 5]
```diff
@@ -61,19 +64,24 @@ RHEL 7.*
...
 
-- If you have a problem building on **RHEL 6** check the :ref:`Pre-requisites
-  <source-install-redhat-prereqs>` to make sure sure you are using a more
-  recent version of :command:`gcc` (minimum required is 4.8)
-
 - curl looks for certificates in :file:`/etc/pki/tls/certs/ca-bundle.crt`
   rather than :file:`/etc/ssl/certs/ca-certificates.crt`. The solution is to
   copy :file:`ca-certificates.crt` to :file:`ca-bundle.crt`.
 
+
```
[the remark this answers is not in the document]
Comment (gcomoretto) [reply known-issues.rst]: No I remove it.

File: known-issues.rst (hunk 1 above)
Comment (jdswinbank) [reply known-issues.rst]: I suggest that there's a potential problem here, which bears mentioning, but is not well described by the current text.

Comment (gcomoretto) [reply known-issues.rst]: OK, I have removed the section.

**Pull request merged**